components/signup/signup.tsx

```diff
@@ -455,16 +458,25 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
         }
     };
 
+    function sendFirstAdminUserTelemetryEvents(errorId: string) {
+        const sbr = getSbr();
+        if (!sbr) {
```

Comment:
I am leaveraging the sbr value (if I understood it correctly, it captures who has invited you to join the workspace) so basically if this value is empty means you are the first admin.

Reply:
I don't think we should rely on the `sbr` to determine first admin as this value could not be present in many use cases, as directly accessing the signup page.

Reply:
I agree that we shouldn't rely on `sbr` to determine first admin and I don't think we have a reliable way to guarantee just the first admin page view event. 

We should track just general page view events for the account creation page and this would be enough to get the visibility we need to see that first event come through if the first admin lands on this page. Would this be enough @esethna ?

Reply:
ok, but I think we can still use the sbr, even if it is not a 100% reliable way of defining a first admin access it would reduced a lot the tracking of first admin users vs normal users accessing the sign up page. What I meanis, what are the chances of users landing on the sign up page by just typing the url and visiting the page directly, versus first admin users landing on the page without an sbr value?